### ISSUE TITLE
Request for training code

### ISSUE THREAD

**Issue by guanhuankang** [Mhaiyang/CVPR2021_PDNet#2]
Thank you for the release of the codebase. I am trying to train your model, but I found that some training details are lack. So I am writing to ask for the training code. Or could you provide the implementation of loss functions and the code for computing metrics (weighted f-measure) ? 

Looking forward to your response! Thanks again!
